src/for_2D_build/meshes/cell_linked_list.hpp

```diff
@@ -45,19 +45,21 @@ void CellLinkedList::searchNeighborsByParticles(
 }
 //=================================================================================================//
 template <class DynamicsRange, typename GetSearchDepth, typename GetNeighborRelation>
-void CellLinkedListKernel::searchNeighborsByParticles(
+ExecutionEvent CellLinkedListKernel::searchNeighborsByParticles(
```

Comment:
No namespace of "execution" here?

Reply:
You are right, it should have it, I'll fix it. Curious how it doesn't throw errors or warnings at compile time. Probably because particle_iterators.h is included in cell_linked_list.hpp, which has the using namespace directive for execution.

Reply:
thx, I approved this PR.